project/V.scala

```diff
@@ -118,13 +118,13 @@ object V {
 
   val quickPublishScalaVersions =
     Set(
-      scala211,
-      sbtScala,
-      scala212,
-      ammonite212Version,
-      scala213,
-      ammonite213Version,
-      scala3,
-      ammonite3Version,
+      // scala211,
+      // sbtScala,
+      // scala212,
+      // ammonite212Version,
+      // scala213,
+      // ammonite213Version,
+      scala3
+      // ammonite3Version,
```

Comment:
```suggestion
      scala211,
      sbtScala,
      scala212,
      ammonite212Version,
      scala213,
      ammonite213Version,
      scala3,
      ammonite3Version,
```

Reply:
Ach, my bad! I removed it for the slow tests to run faster.